src/scripts/main.js

```diff
@@ -1,7 +1,25 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+
+  div.insertAdjacentHTML(
+    'afterbegin',
+    `<h2 class="title">${title}</h2>
+    <p>${description}</p>`,
```

Comment:
There is a syntax error in the HTML string. The closing tag for the `<h2>` element is missing a closing angle bracket. It should be `</h2>` instead of `<h2>`.